partitioned-table.md

```diff
@@ -1699,15 +1694,13 @@ To address these issues, TiDB introduces the global indexes feature in v8.3.0. A
 >
 > The global indexes feature is experimental. It is not recommended that you use it in the production environment. This feature might be changed or removed without prior notice. If you find a bug, you can report an [issue](https://github.com/pingcap/tidb/issues) on GitHub.
```

Comment:
It seems that we should remove this warning about experimental global indexes? I see that this warning is removed in the PR https://github.com/pingcap/docs-cn/pull/18659 for docs in Chinese.

Reply:
Removed already in this PR https://github.com/pingcap/docs/pull/18940/files

Reply:
Got it.